src/v/utils/tests/base64_test.cc

```diff
@@ -49,3 +49,34 @@ BOOST_AUTO_TEST_CASE(iobuf_type) {
     auto decoded = base64_to_bytes(encoded);
     BOOST_REQUIRE_EQUAL(decoded, iobuf_to_bytes(buf));
 }
+
+BOOST_AUTO_TEST_CASE(base64_url_decode_test) {
```

Comment:
Can tests be added for 0, 1, 2, 3 byte decodes, as well as some failures.

Reply:
stellar, will add